cg/store/crud/read.py

```diff
@@ -497,6 +498,25 @@ def get_illumina_sequencing_run_by_device_internal_id(
             device_internal_id=device_internal_id,
         ).first()
 
+    def get_latest_illumina_sequencing_run_for_case(
```

Comment:
So I tried to make this the same / similar to the old implementation as `get_latest_flow_cell_on_case`.

This is for NIPT only, I assume that a single  case is associated to a single sequencing run here. I can make that clear in the docstring. Perhaps we should inidcate that this query is NIPT tailored.